plugin/src/main/java/org/openrewrite/gradle/isolated/DefaultProjectParser.java

```diff
@@ -1303,9 +1305,9 @@ private SourceFileStream parseMultiplatformKotlinProject(Project subproject, Col
                 try {
                     implementationClasspath = rewriteImplementation.resolve();
                 } catch (Exception e) {
-                    logger.warn("Failed to resolve dependencies from {}:{}. Some type information may be incomplete",
-                            subproject.getPath(), implementationName);
-                    implementationClasspath = emptySet();
+                    throw new GradleException(String.format("Failed to resolve the dependencies from %s:%s; " +
+                                    "We can not reliably continue without this information.",
+                            subproject.getPath(), implementationName), e);
```

Comment:
As discussed internally we agreed in principle with throwing an exception; now need to replicate this to the other instances where we call `resolve()`; some of those hadn't yet done any exception handling, so need a bit of time to figure if throwing is even necessary there.

Reply:
So just a question, but is the default Gradle exception not sufficient when resolution fails? In this case, were creating a resolvable configuration where we may have had one that wasn't resolvable.